src/ZonedDateTime.php

```diff
@@ -232,6 +232,12 @@ public static function fromDateTime(\DateTimeInterface $dateTime) : ZonedDateTim
 
         $dateTimeZone = $dateTime->getTimezone();
 
+        /**
+         * @psalm-suppress TypeDoesNotContainType
+         *
+         * Looks like Psalm might have a wrong return type for \DateTimeInterface::getTimeZone - it should
+         * be \DateTimeZone|false . Psalm seems to think it's just \DateTimeZone
+         */
```

Comment:
Reported here: https://github.com/vimeo/psalm/issues/4515
Please add a link in the comment!

Reply:
Done